Controller/SectionController.php

```diff
@@ -46,11 +46,13 @@ public function listAction()
     {
         $sectionList = $this->sectionService->loadSections();
         $contentCountBySectionId = [];
+        $canDeleteBySectionId = [];
```

Comment:
I'd actually agree with Carlos here: we have said that "can" was reserved for permissions. Could you change it to deletable ? Maybe then change it to only contain the ones that are deletable, and check in the template using is defined `{% if deletableSections[section.id] is defined %}`.